databend_sqlalchemy/provision.py

```diff
@@ -36,3 +36,8 @@ def _databend_drop_db(cfg, eng, ident):
 def _databend_configure_follower(config, ident):
     config.test_schema = "%s_test_schema" % ident
     config.test_schema_2 = "%s_test_schema_2" % ident
+
+# Uncomment to debug SQL Statements in tests
```

Comment:
Why contain these comments?

Reply:
I include the comments because it took me a good while to figure out how to show the debug SQL statements. I can remove them from the PR though if required and keep them locally. I figured it might help someone else in the future.

Reply:
Fine